### PR TITLE
inhibit ElasticsearchClusterNotHealthy if its csv is not in success

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

**Comment by yithian** [comment]
depends on https://github.com/openshift/elasticsearch-operator/pull/609

**Comment by yithian** [comment]
/hold

until after change freeze

File: pkg/controller/secret/secret_controller.go [hunk 1]
```diff
@@ -407,6 +407,17 @@ func createAlertManagerConfig(pagerdutyRoutingKey, watchdogURL, consoleUrl strin
 					"alertname": "KubeDeploymentReplicasMismatch",
 				},
 			},
+			{
+				Equal: []string{
+					"dummylabel",
```
Comment (yithian) [inline pkg/controller/secret/secret_controller.go:412]: NB this label obviously won't match and that's both ok and expected. When a label is missing (or empty) on both source and target, the rule will apply (see: [docs](https://www.prometheus.io/docs/alerting/latest/configuration/#inhibit_rule) ).

If there wasn't a label here, the tests exploded spectacularly, so I figured a label that would never match is the next best thing.

Comment (cblecker) [reply pkg/controller/secret/secret_controller.go:412]: Is it possible to get a code comment describing this? I figure this is something we will be scratching our heads about in the future

**Comment by yithian** [comment]
/unhold

this should be fine to land ahead of elasticsearch-operator 4.7

**Pull request merged**